Assets/Scripts/BowlingBall.cs

```diff
@@ -1,151 +0,0 @@
-﻿using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.Security.AccessControl;
-using UnityEngine;
-using UnityEngine.EventSystems;
-using UnityEngine.Experimental.UIElements;
-using UnityEngine.XR.WSA;
-
-public class BowlingBall : MonoBehaviour
-{
-
-	// Use this for initialization
-	public Vector3 InitialVelocity = Vector3.forward;
```

Comment:
You're using a public Vector3 here, seemingly so you can edit at design time.

Unless you also want classes outside of BowlingBall to be fiddling with these values possibly at runtime, it's best practice for them to be:

```[SerializeField] private```

Instead. This will make sure Unity saves the data, and it is available to edit at design time, but can't be changed externally.